README.md

```diff
@@ -19,10 +19,14 @@ We've turned on [discussions](https://github.com/UCL-ARC/python-tooling/discussi
 
 ## Using this template
 
-1. Install [cookieninja]
-   ```
-   pip install cookieninja
-   ```
+1.  Install [cookieninja](https://libraries.io/pypi/cookieninja) in a `conda`, `mamba` or `venv` environment (commented lines for conda example).
+
+```
+# conda create -n ptoolingVE python=3.11 pip -c conda-forge
+# conda activate ptoolingVE
+pip install cookieninja build twine setuptools-scm pytest pre-commit
```

Comment:
Once they've created their project, they should be able to do `python -m pip install -e ".[dev]"` rather than installing them manually (as these packages are listed as [optional dependencies](https://github.com/UCL-ARC/python-tooling/blob/cb5e1cbd948c9c0276a901b2eaaa9b17208b1021/%7B%7Bcookiecutter.project_slug%7D%7D/pyproject.toml#L33-L41))

Reply:
Can we not pin the python version? The more we pin, the more we have to change later.

Agree regarding `dev`.

You also don't need to install `setuptools-scm`. Plus, we are using `tox` so no need to install `pytest`.